src/alert/styles.scss

```diff
@@ -76,6 +68,9 @@
 }
 
 .alert-focus-wrapper {
+  flex: 1;
+  min-inline-size: 70%; // fallback for browsers that don't support calc-size
+  min-inline-size: calc-size(max-content, min(size, 70%));
```

Comment:
Looks like a non-standard feature yet: https://developer.mozilla.org/en-US/docs/Web/CSS/calc-size

> Experimental. Expect behavior to change in the future.


Leaving this up to your consideration if there could be breaking changes in this feature, making it work differently in newer browser releases

Reply:
I tried to understand CSS standardization process, but could not really figure how close is this spec to finalization 🤷

Reply:
agree it's not entirely clear, but I think the relatively limited usage of it here makes the risk acceptable